AsyncDisplayKit/ASCollectionView.h

```diff
@@ -120,4 +128,32 @@
 - (void)collectionView:(ASCollectionView *)collectionView willDisplayNodeForItemAtIndexPath:(NSIndexPath *)indexPath;
 - (void)collectionView:(ASCollectionView *)collectionView didEndDisplayingNodeForItemAtIndexPath:(NSIndexPath*)indexPath;
 
+/**
+ * Tell the collectionView if batch fetching should begin.
+ *
+ * @param collectionView The sender.
+ *
+ * @discussion Use this method to conditionally fetch batches. Example use cases are: limiting the total number of
+ * objects that can be fetched or no network connection.
+ *
+ * If not implemented, the collectionView assumes that it should notify its asyncDelegate when batch fetching
+ * should occur.
+ */
+- (BOOL)shouldBatchFetchForCollectionView:(ASCollectionView *)collectionView;
```

Comment:
`-shouldBatchFetchForCollectionView:` versus `-collectionView:beginBatchFetchingWithContext:` is a little confusing, especially since this method is declared before that one — I can imagine someone getting mixed up and actually executing the batch fetch here.  No strong objections, but maybe we could tweak the name to be more unambiguous?

Or, maybe we could remove this method and have people call `[context cancelBatchFetching]` if they're not yet ready to fetch something?  That's clunky, though.